opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -1481,3 +1484,12 @@ def test_span_no_limits_code(self):
                 )
             )
         )
+
+    def test_dropped_attributes(self):
+        span = get_span_with_dropped_attributes_events_links()
+        self.assertEqual(1, span.dropped_links)
+        self.assertEqual(2, span.dropped_attributes)
+        self.assertEqual(3, span.dropped_events)
+        self.assertEqual(2, span.events[0].attributes.dropped)
+        self.assertEqual(2, span.links[0].attributes.dropped)
+        self.assertEqual(2, span.resource.attributes.dropped)
```

Comment:
Hmm, are dropped attributes from events, links added to total dropped count or dropped_attributes_count is more like dropped_span_attributes_count?

Reply:
Dropped attributes counts are independent for events/links/spans/resources.

Reply:
Correct. My question is whether dropped attributes counts of events/links are reported for exporters?

Reply:
Dropped attribute counts for events/links are reported for OTLP as defined in [proto](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/trace/v1/trace.proto#L191). I don't think they have been defined for the other exporters.

Reply:
I believe we are not doing that in OTLP exporter https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py#L169.

Reply:
Correct, I noted that in this [comment](https://github.com/open-telemetry/opentelemetry-python/pull/1893#discussion_r646996075). I believe we should add this as well.

Reply:
I added them now, please take a look

Reply:
Note that according to the spec, the Zipkin exporter does not provide a mechanism to export attributes for events, and support for Links is TBD.

The Jaeger exporter described in the spec suggests implementations MAY support attributes for links via logs, but doesn't require it.